lattice-controller/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasmcloud-provider-lattice-controller"
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
`0.3.0` hasn't been released yet so I think it's OK to not bump this at all yet, I bumped to `0.3.0` with a breaking change anticipating this would be included